github/ipfs.yml

```diff
@@ -6530,24 +6432,13 @@ teams:
     privacy: closed
   go-ipfs release:
     description: Who may cut a release of go-ipfs
-    members:
-      member:
-        - aschmahmann
-        - guseggert
-        - lidel
-        - olizilla
-        - petar
-        - Stebalien
     privacy: closed
```

Comment:
I think this entire group can be removed, releases were done by people who were not members.

Reply:
Let's remove empty teams after the user related changes are complete.